packages/sdk-js/src/util.ts

```diff
@@ -62,6 +62,42 @@ export function getUrlRegExp(regexString: string): RegExp | undefined {
   }
 }
 
+export function isURLTargeted(url: string, targets: UrlTarget[]) {
+  if (!targets.length) return true;
+  let hasIncludeRules = false;
+  let isIncluded = false;
+
+  for (let i = 0; i < targets.length; i++) {
+    const match = _evalURLTarget(url, targets[i].type, targets[i].pattern);
+    if (targets[i].include === false) {
+      if (match) return false;
+    } else {
+      hasIncludeRules = true;
+      if (match) isIncluded = true;
```

Comment:
wouldn't this just be `return true` and then skip the final check for `isIncluded || !hasIncludeRules`

Reply:
@bryce-fitzsimons is this blocking?